Glyssen/Quote/QuoteParser.cs

```diff
@@ -42,6 +42,28 @@ public static void ParseProject(Project project, BackgroundWorker projectWorker)
 			projectWorker.ReportProgress(100);
 		}
 
+		internal static List<BookScript> TestQuoteSystem(Project project, QuoteSystem altQuoteSystem)
```

Comment:
Unless I am missing something, it seems that this test parse starts with the books/blocks in their current state.  We need to parse from a fresh (unparsed) state.  The discussed strategy means writing some kind of Unparse algorithm to reverse engineer the parse back to the original state.